packages/client/src/pages/ContentPage/index.tsx

```diff
@@ -1,20 +1,20 @@
-import { memo, ReactNode } from 'react';
 import { Avatar, Skeleton } from 'antd';
+import { memo, ReactNode } from 'react';
 import { useTranslation } from 'react-i18next';
 import { useNavigate } from 'react-router-dom';
 
+import ApiIcon from '@/assets/svgs/api.svg?react';
 import ProjectIcon from '@/assets/svgs/project.svg?react';
 import RunIcon from '@/assets/svgs/run.svg?react';
 import TokenIcon from '@/assets/svgs/token.svg?react';
-import ApiIcon from '@/assets/svgs/api.svg?react';
```

Comment:
Please remove unnecessary changes. If we want imports ordered automatically, we should configure ESLint or Prettier accordingly.